packages/boxel-ui/addon/helpers/truth-helpers.ts

```diff
@@ -10,7 +10,7 @@ export function gt<T>(a: T, b: T): boolean {
   return a > b;
 }
 
-export function and<T>(...args: [T, T, ...T[]]): boolean {
+export function and<T>(...args: [T | any, T | any, ...T[]]): boolean {
```

Comment:
i don't agree with this type change. it sounds like what you want to do is check for truthyiness in addition to and-ing. in that case please nest with the `bool` helper

Reply:
Thanks for the feedback! I didn't see the `bool` helper before. Updated [2281a83](https://github.com/cardstack/boxel/pull/616/commits/2281a83db15dd12a30e243f1082ca01539c0867a)